bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -275,14 +196,12 @@ spec:
         hostPath:
           path: /var/lib/openvswitch/etc
       - name: run-openvswitch
-        emptyDir: {}
+        hostPath:
+          path: /run/openvswitch
```

Comment:
Could you verify that this is indeed the path when the OVS socket lives when OVS is installed on RHEL7 from an RPM?

Reply:
Yes
Note also that /var/run/openvswitch is a link to /run/openvswitch.